lib/Runtime/PlatformAgnostic/RuntimePlatformAgnosticPch.h

```diff
@@ -7,7 +7,7 @@
 #ifdef _WIN32
 #include "CommonMin.h"
 #else
-#include "pal.h"
+#include "CommonPal.h"
```

Comment:
?

Reply:
Utf8Helper.h uses wcslen, so it has to be available _and_ properly `#define`d to PAL_wcslen on xplat. Simply including pal.h didn't get that done, since there needs to be a combination of #defines in order to make that work. I could try to recreate the defines, but my initial stab caused other errors on Linux, and including CommonPal.h seemed easier (and it matches what we do on win32, where we include CommonMin.h)